Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM tiangolo/uwsgi-nginx:python3.6-alpine3.7
```

Comment:
I would like to move away from this Dockerhub image. The plan is to build my own and host it in my dockerhub to ensure that we have more control over the image.